docs/reference/resp2-to-resp3-replies.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "RESP2 to RESP3 replies migration guide"
+linkTitle: "RESP2 to RESP3 migration"
+weight: 4
+description: RESP2 to RESP3 replies reference for clients developers
+aliases:
+    - /topics/resp2-to-resp3-replies/
+---
+
+In the journey from RESP2 to [RESP3](https://redis.io/docs/reference/protocol-spec/), one of the primary motivations was 
```

Comment:
IMHO we shouldn't do any manual line breaking, but rather break at periods, see the README in the repo

Reply:
Fixed